addons/website/views/website_templates.xml

```diff
@@ -1775,7 +1775,7 @@
                  tabindex="-1"
                  role="dialog">
                 <div class="modal-dialog d-flex s_popup_size_full">
-                    <div class="modal-content oe_structure">
+                    <div class="modal-content">
```

Comment:
I don't think you should entirely prevent to drop snippet in the cookie bar, it might make sense when it's layout is the "popup" one... we could prevent that for the "discrete" mode though, maybe with use of data-exclude

Reply:
In the new version I added a line of code where we add `oe_structure` if we switch to the popup layout, would that be an acceptable way to keep functionality?

Reply:
That's another solution I guess :+1: